src/main/java/no/nav/fo/veilarbregistrering/registrering/bruker/PubliseringAvEventsService.java

```diff
@@ -0,0 +1,66 @@
+package no.nav.fo.veilarbregistrering.registrering.bruker;
+
+import no.nav.fo.veilarbregistrering.bruker.Bruker;
+import no.nav.fo.veilarbregistrering.profilering.Profilering;
+import no.nav.fo.veilarbregistrering.profilering.ProfileringRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+
+public class PubliseringAvEventsService {
```

Comment:
Denne er litt premature - skulle egentlig ikke med, men er tenkt å overta ansvaret for å publisere Kafka-events for arbeidssøker registrering og profilert.